neusomatic/include/Options.h

```diff
@@ -45,7 +46,8 @@ namespace neusomatic {
     std::cerr<< "-o/--out_count_file,                  output count file path,                                                required.\n";
     std::cerr<< "-w/--window_size,                     window size to scan the variants,                                      default is 15\n";
     std::cerr<< "-y/--fully_contained,                 if this option is on. A read has to be fully contained in the region,  default is False\n";
-    std::cerr<< "-t/--num_threads,                      numbef or thread used for building the count matrix,                   default is 4\n";
+    std::cerr<< "-t/--num_threads,                     numbef or thread used for building the count matrix,                   default is 4\n";
+    std::cerr<< "-d/--max_depth,                       maximum depth for building the count matrix,                           default is 40,000\n";
```

Comment:
Do we parse commas in the option? If not, better to change "40,000" to "40000" to avoid confusion.

Reply:
That is only for display. This string is not parsed.